src/data/api/ApiRequest.ts

```diff
@@ -4,36 +4,56 @@ import ActiveApiEndpoint from '../ActiveApiEndpoint';
 import Config from 'react-native-config';
 import { api } from '../consts.json';
 
-function getRequest(
+async function getRequest(
   baseEndpoint: string,
   endpoint: string,
   endpointParams: Object = {},
   body: Object = {}
-): Promise<AxiosResponse> {
+): Promise<AxiosResponse | null> {
   const formattedEndpoint = Mustache.render(endpoint, endpointParams);
-  return axios.get(`${ActiveApiEndpoint()}/${baseEndpoint}/${formattedEndpoint}`, {
-    headers: {
-      [api.headers.authKey.key]: Config.API_KEY,
-      'Content-Type': 'application/json',
-    },
-    data: body,
-  });
+
+  return await axios
```

Comment:
Kan dit niet gewoon terug zoals het eerst was?

Reply:
Er kwamen anders honderden warnings een unhandled Promise. Dit is én netter en zorgt niet meer voor warnings.